src/NUnitEngine/nunit.engine/Drivers/NUnit3FrameworkDriver.cs

```diff
@@ -80,7 +80,7 @@ public string Load(string testAssemblyPath, IDictionary<string, object> settings
             }
             catch (SerializationException ex)
             {
-                throw new NUnitEngineException("The NUnit 3.0 driver does not support the portable version of NUnit. Use a platform specific runner.", ex);
+                throw new NUnitEngineException("The NUnit 3.0 driver cannot support this test assembly. Use a platform specific runner.", ex);
```

Comment:
The old error message was actually being triggered by Silverlight and CF builds as well as portable, so it has been reworded. It is now an exception of last resort so it's still appropriate to terminate execution here. Once we are this far in the code, it's hard to do anything else.